src/contentmanagerdelegate.cpp

```diff
@@ -195,9 +195,17 @@ void ContentManagerDelegate::paint(QPainter *painter, const QStyleOptionViewItem
     if (index.parent().isValid()) {
         // additional info
         QRect nRect = r;
-        auto viewWidth = KiwixApp::instance()->getContentManager()->getView()->getView()->width();
-        nRect.setWidth(viewWidth);
-        painter->drawText(nRect, Qt::AlignLeft | Qt::AlignVCenter, index.data(Qt::UserRole+1).toString());
+        auto view = KiwixApp::instance()->getContentManager()->getView()->getView();
+
+        /* Align text to the x coordinate of column index 1, the Name column. */
+        nRect.setX(view->visualRect(index.parent().siblingAtColumn(1)).x());
+        nRect.setWidth(view->width());
+
+        /* Enable word wrap here. QTreeView word wrap doesn't apply on drawText. */
+        QTextOption op;
+        op.setAlignment(Qt::AlignLeft | Qt::AlignVCenter);
+        op.setWrapMode(QTextOption::WordWrap);
+        painter->drawText(nRect, index.data(Qt::UserRole+1).toString(), op);
```

Comment:
Extract this piece of code into a new method `ContextManagerDelegate::paintBookDescription()`